pysces/body.py

```diff
@@ -48,11 +49,78 @@ def cylinder(radius, num_points):
     return Body(points)
 
 def flat_plate(num_points):
-    """Return a flat plate with the given number of points"""
-    x = np.linspace(1, 0, num_points)
+    """
+    Return a flat plate with the given number of points.  In body coordinates
```

Comment:
Would be good to make these doc strings consistent with [PEP 257](https://www.python.org/dev/peps/pep-0257/): either a single line, or a single line, followed by a blank line, followed by more details.  This makes the formatting look good in things like iPython and Sphinx.